modules/ossm-rn-known-issues.adoc

```diff
@@ -19,6 +19,8 @@ These limitations exist in {ProductName}:
 
 * Graph layout - The layout for the Kiali graph can render differently, depending on your application architecture and the data to display (number of graph nodes and their interactions). Because it is difficult if not impossible to create a single layout that renders nicely for every situation, Kiali offers a choice of several different layouts. To choose a different layout, you can choose a different *Layout Schema* from the *Graph Settings* menu.
 
+* The first time you access related services (Jaeger, Grafana) from the Kiali console, you must accept the certificate and re-authenticate using your OpenShift login credentials.  This happens due to an issue with how the framework displays embedded pages in the Console.
```

Comment:
ISG suggests avoiding parentheses. Suggestion: "The first time you access related services, such as Jaeger or Grafana, from the Kiali console..."
Also: 
- `OpenShift` -> `{product-title}` 
- There's a double space between the two sentences, not sure if that is rendered in output
- Console -> console (as lowercased in the first sentence)